### DISCUSSION TITLE
Stop FastLED Matrix from scrolling<title>

### DISCUSSION THREAD

**Discussion by billz-net** [community/community#112572] [Programming Help]
### Select Topic Area

Product Feedback

### Body

I’ve watched so many FastLED videos, but, none have addressed the simple effect that I’m looking for and I can’t seen to get it right.

For my HAM opperator friends, I want to make a 32X8 Matrix sign that will scroll from right to left their call sign till it gets centered in the matrix, then stop for 60 seconds.  Then blank out, and scroll from left to right till it centers and stop for 60 seconds.  The same from bottom to top & from top to bottom.  This will go on and on for ever (with maybe some color changes).

It is the stop scrolling that really has me stumped.  Nothing is working so I don't have any sample code except for the default scrolling that does work.

Probably simple but I need some help.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @billz-net ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by nadila2k** [comment]
response #1
